### DISCUSSION TITLE
A95X and GENERIC S905

### DISCUSSION THREAD

**Discussion by fachos** [devmfc/debian-on-amlogic#114] [General]
Hi devmfc,

trying install to nexbox a95x 1g/8g which is s905 chipset....
bootup is successful.. installation is smooth.. but after reboot/poweroff
it didn't show anything on screen... try put back the  sdcard and fire up also 
nothing happen...  and I have to flash back to factory firmware to make this box alive again...
few times again and again still same result, maybe you can help me how to solve this?

thank you

**Comment by iisnsb** [comment]
Maybe you need to connect it on SSH. If you are using mobile phone,then you can use ServerBox to connect it.And if you are using Windows PC,then you can use WinSCP to connect.Of corse,for connecting,you must know the IP address of your    
 Tvbox.You can find it on Router background management.Good luck.

**Reply by fachos** [reply]
Hi @iisnsb 

i can bootup and SSH to the box.... the thing is i cant install to the internal storage
after the installation script done i poweroff and takeout sdcard.. power it back and nothing happen 
seem like box is bricked. i have to flash the android OEM firmware back to bring it alive..
i have few different box with s905x/s905w/s912 chipset can install them without any problem... 

thank for respond me

**Reply by iisnsb** [reply]
Oh.I think you need to run aml-install-to-emmc.sh And then you have to run aml-multiboot-setup.sh 
Of course, you can find them in /root.

**Reply by devmfc** [reply]
it's the other way around, first  ./aml-multiboot-setup.sh, than ./aml-install-to-emmc.sh ;-)

But anyway, you can't install anything to emmc on a S905 with a standard partition table. That is because sector 0 is used as starting point for the bootrom to load the bootloader from. If you write to sector 0 (where the normal partition tables live)  you (soft)brick the box.

**Comment by fachos** [comment]
thank you anyway.... i just hammer the box... useless s905

**Reply by iisnsb** [reply]
All right.Maybe you can try Armbian for Amlogic https://github.com/ophub/amlogic-s9xxx-armbian?tab=readme-ov-file.Good luck.
